artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
Just make it true or false every time.  And make it mandatory. 

The three arguments make it confusing for the user.